ariadne/src/Ariadne/Config/CLI.hs

```diff
@@ -276,7 +275,7 @@ opts :: FilePath -> Opt.ParserInfo (FilePath, Bool, CLI_AriadneConfig)
 opts xdgConfigPath = Opt.info ((parseOptions xdgConfigPath) <**> Opt.helper)
   (  Opt.fullDesc
   <> Opt.header "Ariadne wallet"
-  <> Opt.footer ("For more details see " <> toString ariadneURL))
+  <> Opt.footer ("For more details see https://serokell.io/ariadne/"))
```

Comment:
This is the only unfortunate copy-paste of the link, but the previous "cross-backend" dependency was even worse.

Reply:
Maybe let's define `defaultAriadneUrl = "https://serokell.io/ariadne/"` in `Ariadne.Config.Update` and use it here and in `defaultUpdateConfig`?

Reply:
The URL here is definitely not related to update system, and in principle shouldn't be configured at all.

Reply:
Imo, that's conceptually exactly the same URL as displayed by update system. Update system just tells you to go to the main website of the wallet, not to the place where it can be downloaded. Help message says the same :man_shrugging:

Reply:
No, update system could potentially provide a direct URL to release page with version tag or something.
This one here is more of URL for a support page.